src/main/g8/project/ProjectPlugin.scala

```diff
@@ -0,0 +1,10 @@
+import freestyle.FreestylePlugin
+import sbt._
+import sbtorgpolicies.OrgPoliciesPlugin.autoImport._
+object ProjectPlugin extends AutoPlugin {
+  override def requires: Plugins = FreestylePlugin
+
+  override def trigger: PluginTrigger = allRequirements
+
+  override def projectSettings: Seq[Def.Setting[_]] = scalaMetaSettings
+}
```

Comment:
I'd suggest adding `scalaMetaSettings` below this line: https://github.com/frees-io/freestyle-template.g8/blob/arf-Update-sbr-freestyle-version/src/main/g8/build.sbt#L8